### PR TITLE
Contacts listing page

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by naufil50**
Fixes #4 

contacts-ui-1

contacts-ui-2

### CODE REVIEW

File: app/javascript/src/components/Common/Sidebar/constants.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { UserCircle as Contacts } from "@bigbinary/neeto-icons";
 import { Notes, Settings } from "neetoicons";
```
Comment (yedhink) [inline app/javascript/src/components/Common/Sidebar/constants.js:2]: Why are there two different styles of import? Stick onto one of them. Preferably the latter one.

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { Table } from "@bigbinary/neetoui";
+
+import { CONTACTS_COLUMNS as columns } from "./constants";
+import { CONTACTS_ROWS as rows } from "./constants";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:6]: - Why are you renaming literal constants to lowercase names?
- Why are you not importing all entities in a single statement from same module? What's the point of splitting them out into two separate import statements?

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function ContactsList() {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:8]: https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <Table
+      columnData={columns}
+      currentPageNumber={1}
+      defaultPageSize={10}
+      handlePageChange={function noRefCheck() {}}
+      onRowClick={function noRefCheck() {}}
+      onRowSelect={function noRefCheck() {}}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:16]: https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+const Contacts = () => {
+  const [showMenu, setshowMenu] = useState(true);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    fetchContacts();
+  }, []);
+
+  const fetchContacts = () => {};
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:17]: Is this useEffect required? Does it add any value to the current page?

**Review by yedhink — COMMENTED** [review]
@naufil50 Few comments. Refer other PRs too for review comments.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx (hunk 5 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Contacts/index.jsx:17]: Yes removed. thanks

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx (hunk 4 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:16]: Done. Thanks.

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx (hunk 3 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:8]: Done. Thanks

File: app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx (hunk 2 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Contacts/ContactsList.jsx:6]: Sure, Done changes. Thanks

File: app/javascript/src/components/Common/Sidebar/constants.js (hunk 1 above)
Comment (naufil50) [reply app/javascript/src/components/Common/Sidebar/constants.js:2]: This is was corrected afterwards.

**Pull request merged**